tests/util/simple_tests.rs

```diff
@@ -99,9 +99,9 @@ pub fn test_force_flag_stratis(paths: &[&Path]) -> () {
     initialize(&uuid, unique_devices.clone(), MIN_MDA_SECTORS, false).unwrap();
     assert!(initialize(&uuid2, unique_devices.clone(), MIN_MDA_SECTORS, false).is_err());
 
-    // FIXME: once requirement that number of devices added be at least 2 is removed
-    // this should succeed.
-    assert!(initialize(&uuid, unique_devices.clone(), MIN_MDA_SECTORS, false).is_err());
+    // Attempting to initialize existing blockdevs into the pool they
```

Comment:
Feel free to drop these comment lines, since the header is fixed. They're unnecessary.